variables.tf

```diff
@@ -892,6 +892,12 @@ variable "codepipeline_s3_bucket_force_destroy" {
   default     = false
 }
 
+variable "codepipeline_s3_cache_type" {
```

Comment:
Variable name should be `codebuild_cache_type`.
I think defaulting to `NO_CACHE` will be best, otherwise you will hit the same error you encountered. I think we need to test this PR with values `S3` and `NO_CACHE`.

Reply:
@joe-niland - thanks, I'll make the change and set the default to `NO_CACHE`. Actually, I have been testing with NO_CACHE locally, and seems to have resolved the original problem I had messaged in slack about.